common/config/azure-pipelines/jobs/extract-api.yaml

```diff
@@ -0,0 +1,38 @@
+# iModel.js Extract API Build
+
+trigger: none
+
+schedules:
+  - cron: "0 5 * * *"
+    displayName: Daily midnight build
+    branches:
+      include:
+        - master
+
+pool:
+  vmImage: ubuntu-latest
+
+jobs:
+- job:
+  steps:
+    - checkout: self
+      clean: true
+    - task: NodeTool@0
+      displayName: "Use Node 14"
+      inputs:
+        versionSpec: 14.17.0
+        checkLatest: true
+    - script: node common/scripts/install-run-rush.js install
+      displayName: rush install
+      workingDirectory: $(System.DefaultWorkingDirectory)
+    - script: node common/scripts/install-run-rush.js build -v -p max
```

Comment:
Lets use `compile` here instead of `build`. It should be quite a bit faster and I don't think there is anything in `build` necessary.